src/content/advanced/storage/efs/index.md

```diff
@@ -59,9 +59,9 @@ To install the EFS CSI driver in the workload cluster, you will need to follow t
 4. Select the Giant Swarm catalog.
 5. Select the App named `aws-efs-csi-driver`.
 6. Click the Configure & Install button. Make sure that the correct cluster is selected.
-7. Create and submit a configuration values file in case you want to deploy the storage class and controller (by default disable to allow a smooth transitation to CSI driver). Do not forget to replace the `fileSystemID` with the one you created in the previous step, and the AWS account ID in the role ARN. The role is pre-created and ready to use.
+7. Create and submit a configuration values file in case you want to deploy the storage class and controller (by default disable to allow a smooth transitation to CSI driver). Do not forget to replace the `fileSystemID` with the one you created in the previous step, the AWS account ID in the role ARN, and the cluster name in IAM role. The role is pre-created and ready to use.
```

Comment:
```suggestion
Create and submit a configuration values file in case you want to deploy the storage class and controller (by default disabled to allow a smooth transition to CSI driver). Do not forget to replace the `fileSystemID` with the one you created in the previous step, the AWS account ID in the role ARN, and the cluster name in the IAM role. The role is pre-created and ready to use.
```